frontend/src/pages/LogIn/index.jsx

```diff
@@ -48,37 +38,35 @@ export default function LogIn() {
   }
   return (
     <SSwitchTabs>
-      <ThemeProvider theme={theme}>
-        <div className="tab">
-          <Box sx={{ width: "100%" }}>
-            <Box sx={{ borderBottom: 1, borderColor: "divider" }}>
-              <Tabs
-                centered
-                value={value}
-                onChange={handleChange}
-                aria-label="basic tabs example"
-              >
-                <Tab
-                  label="Se connecter"
-                  id={tabIndex(0).id}
-                  aria-controls={tabIndex(0)["aria-controls"]}
-                />
-                <Tab
-                  label="S'inscrire"
-                  id={tabIndex(1).id}
-                  aria-controls={tabIndex(1)["aria-controls"]}
-                />
-              </Tabs>
-            </Box>
-            <TabPanel value={value} index={0}>
-              <SignIn />
-            </TabPanel>
-            <TabPanel value={value} index={1}>
-              <SignUp />
-            </TabPanel>
+      <div className="tab">
+        <Box sx={{ width: "100%" }}>
+          <Box sx={{ borderBottom: 1, borderColor: "divider" }}>
+            <Tabs
+              centered
+              value={value}
+              onChange={handleChange}
+              aria-label="basic tabs example"
```

Comment:
At least **pretend** you're not copy-pasting and write something useful ? :smirk: